pallets/loans/src/entities/pricing/external.rs

```diff
@@ -67,6 +72,11 @@ pub struct ExternalPricing<T: Config> {
 
 	/// Reference price used to calculate the interest
 	pub notional: T::Balance,
+
+	/// Maximum variation between the settlement price chosen for
+	/// borrow/repay and the current oracle price.
+	/// See [`ExternalAmount::settlement_price`].
+	pub max_price_variation: T::PerThing,
```

Comment:
This will limit us to a variation between `0-100%`. Is this wanted?

Reply:
Is this wanted (and required), just for that, we've chosen that type.

Reply:
I think I had the wrong mental model. In my mind the variation couldn't be more than 100%, but I guess it can (if oracle price is 1.0 and settlement price is 3.0, the variation is 200%). So then maybe it is better to use a `Rate`

Reply:
But does that use case make sense? I thought that variation should be a minor correction over the oracle price

Reply:
Well, even though that use case is "not real" maybe it makes more sense a `Rate`...

Reply:
Yeah I think it's unlikely but theoretically possible